src/utils.ts

```diff
@@ -26,12 +26,27 @@
 
 module powerbi.extensibility.visual {
     export module wordCloudUtils {
+        const crypto: Crypto = window.crypto || (<any>window).msCrypto;
         export function getRandomColor(): string {
-            const red: number = Math.floor(Math.random() * 255),
-                green: number = Math.floor(Math.random() * 255),
-                blue: number = Math.floor(Math.random() * 255);
+            const array = new Uint8Array(3);
+            crypto.getRandomValues(array);
+            return `rgb(${array.join(",")})`;
+        }
+
+        export function getSecureRandomUint(): number {
+            const buf = new Uint8Array(4);
+            const dv = new DataView(buf.buffer);
+            crypto.getRandomValues(buf);
```

Comment:
Do we really need to use secured generator? I think it doesn't make any sense. We just generate colors and angle. There's no security risk.